pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.black]
 line-length = 88
-target-version = ['py36']
+target-version = ['py38']
```

Comment:
let's do this in a separate PR, it think this will cause other changes to repo due to quality

Reply:
Removed!